site/_config.yml

```diff
@@ -25,9 +25,8 @@ pulsar_repo: https://github.com/apache/incubator-pulsar/tree/master
 baseurl: /
 destination: ../generated-site/content
 
-versions:
-- 1.18
-latest: 1.18
+current_version: 1.19.0-incubating
```

Comment:
does it mean on next new release: we only need to change this file with new version?